packages/lib/test/src/utils/Logger.test.js

```diff
@@ -11,20 +11,20 @@ import {
   LogType,
 } from '../../../src/utils/Logger';
 
-describe('Loggger', function() {
-  afterEach('restore logger', function() {
-    Loggy.silent(true);
+describe('Logger', function() {
+  beforeEach(function () {
+    Loggy.testing(false);
+  });
+
+  afterEach('restore logger', function() {    
+    Loggy.testing(true);
+    Loggy.silent(false);
     Loggy.verbose(false);
     Loggy.logs = {};
     sinon.restore();
   });
 
   describe('default attributes', function() {
-    it('is silent and non-verbose by default', function() {
-      Loggy.isSilent.should.be.true;
-      Loggy.isVerbose.should.be.false;
-    });
-
     it(`has a 'logs' object initialized`, function() {
```

Comment:
This test seems to be failing because `Loggy.logs` is being filled by some other test. I believe it's not even necessary to have it, as it's only testing a variable assignment rather than a behavior.